### PR TITLE
Dashboard UI

### PR DESCRIPTION
**Pull request by RoopashreeLakshmikeshava**
1) Added mat-grid-list.
2) Added date range picker(33-UI)
3) Removed inline styles.

### CODE REVIEW

File: src/app/modules/dashboard/dashboard-view/dashboard-view.component.html [hunk 1]
```diff
@@ -1,14 +1,49 @@
...
+                <mat-date-range-input [rangePicker]="picker">
+                  <input matStartDate placeholder="Start date">
+                  <input matEndDate placeholder="End date">
+                </mat-date-range-input>
+                <mat-hint>MM/DD/YYYY – MM/DD/YYYY</mat-hint>
+                <mat-datepicker-toggle matIconSuffix [for]="picker"></mat-datepicker-toggle>
+                <mat-date-range-picker #picker></mat-date-range-picker>
+              </mat-form-field>
+        </div>
+        <br/>
```
Comment (devendra-tarento) [inline src/app/modules/dashboard/dashboard-view/dashboard-view.component.html:24]: @RoopashreeLakshmikeshava  Avoid using br tag , rather using margin and padding for spacing

File: src/app/modules/dashboard/dashboard-view/dashboard-view.component.html [hunk 2]
```diff
@@ -1,14 +1,49 @@
...
+        </mat-grid-list>
+          <mat-grid-list cols="6" rowHeight="10rem" [gutterSize]="'100px'">
+            <mat-grid-tile colspan="4" rowspan="1">
+                <app-shared-table class="table1" [hasFilterOptions]="false" [tableColumns]="dashboardAnalyticsColumns" [tableData]="dashboardAnalyticsData" [isPageable]="false"></app-shared-table>
+            </mat-grid-tile>
+            <mat-grid-tile colspan="2" rowspan="1">
+                <app-shared-table class="table2" [hasFilterOptions]="false" [tableColumns]="dashboardAnaytics2Columns" [tableData]="dashboardAnalytics2Data" [isPageable]="false"></app-shared-table>
+            </mat-grid-tile>
+        </mat-grid-list>
+        <br/>
```
Comment (devendra-tarento) [inline src/app/modules/dashboard/dashboard-view/dashboard-view.component.html:43]: @RoopashreeLakshmikeshava  Avoid using br tag , rather using margin and padding for spacing

File: src/styles.scss [hunk 3]
```diff
@@ -35,3 +35,9 @@ body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
  }
 }
 
+#dashboard {
```
Comment (devendra-tarento) [inline src/styles.scss:38]: @RoopashreeLakshmikeshava  Use dashboard specific styles in dashboard component scss file and also avoid using important keyword until u can not apply styles with by increasing css specificity...

Comment (RoopashreeLakshmikeshava) [reply src/styles.scss:38]: @devendra-tarento , angular material related style could not be applied when I tried adding it at component level. Hence I gave an ID to dashboard component and added the style in global styles file.

File: src/app/modules/dashboard/dashboard-view/dashboard-view.component.html (hunk 2 above)
Comment (RoopashreeLakshmikeshava) [reply src/app/modules/dashboard/dashboard-view/dashboard-view.component.html:43]: Fixing it.

File: src/app/modules/dashboard/dashboard-view/dashboard-view.component.html (hunk 1 above)
Comment (RoopashreeLakshmikeshava) [reply src/app/modules/dashboard/dashboard-view/dashboard-view.component.html:24]: Fixing it

**Pull request merged**